code/game/objects/items/crab17.dm

```diff
@@ -147,42 +152,71 @@
 	START_PROCESSING(SSfastprocess, src)
 
 /obj/structure/checkoutmachine/Destroy()
-	stop_dumping()
 	STOP_PROCESSING(SSfastprocess, src)
+	var/datum/bank_account/crab_account = bogdanoff.get_bank_account()
+	for(var/datum/bank_account/B in SSeconomy.bank_accounts)
+		B.withdrawDelay = 0
```

Comment:
Note that multiple checkout machines can be deployed at the same time, and this would allow you to withdraw